src/ImageRunModal.jsx

```diff
@@ -996,7 +996,7 @@ export class ImageRunModal extends React.Component {
                                     </Flex>
                                 </FormGroup>
                             }
-                            {((userLingeringEnabled && userPodmanRestartAvailable) || (this.isSystem() && podmanRestartAvailable)) &&
+                            {((userLingeringEnabled && userPodmanRestartAvailable) || (this.isSystem())) &&
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2269-fd2cce41-20250919-162110-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)